brave-instrumentation/aws-java-sdk-v2-core/pom.xml

```diff
@@ -74,7 +74,7 @@
     </dependency>
     <dependency>
       <groupId>software.amazon.awssdk</groupId>
-      <artifactId>netty-nio-client</artifactId>
+      <artifactId>url-connection-client</artifactId>
```

Comment:
later, we'll probably want to resurrect this, without deleting the sync side. I noticed that the code paths are different and so there may be bugs on one side (async) and not the other (sync)